upgrading/extending-daml-models/2_new_package.md

```diff
@@ -30,13 +38,6 @@ A package is simply added by pointing to file location of its `dar` (**D**AML **
 When you build a `dar` package with `daml build` within a project without an additional output
 argument, the `dar` will be under the path `.daml/dist/` relativ to your project root.
 
-Remove the file `daml/Main.daml`. This was added as a starting point,
-but we have our own ideas.
-
-```
-rm daml/Main.daml
```

Comment:
I think the best option is to just add a `daml build` after this one. As long as you do that before opening Forum.daml things should be fine. I cannot test it anymore because katacode broke completely for me now (the create-daml-app directory isn’t even created anymore) but I’m reasonably confident.

Reply:
@cocreature I've added the command and have altered the flow. Think this should work better